subscriber_topic: /lane_control/control_signal

publisher_topic: /board_connection/control_pos

subs_queue_size: 1
pubs_queue_size: 1
buff_size: 52428800
default_sobel_kernel_size: 7

img_size: [376, 672]

# Values for Region of Interest (ROI) Cutter
left_bottom: [20, 376]
right_bottom: [652, 376]
apex1: [310, 140]
apex2: [350, 140]
inner_left_bottom: [40, 376]
inner_right_bottom: [336, 376]
inner_apex1: [340, 480]
inner_apex2: [320, 480]

color_config:
  dilatation_kernel_size: 3
  canny_threshold: [80, 200]
  hough_threshold: 20
  hough_min_line_length: 3
  hough_max_line_gap: 1
  sobel_threshold: 40

  hsv_white1:  [0,0,150]
  hsv_white2:  [180,50,255]
  hls_white1:  [0, 200, 0]
  hls_white2:  [180, 255, 255]
  hsv_yellow1: [25,120,90]
  hsv_yellow2: [45,255,255]
  hls_yellow1: [15, 38, 115]
  hls_yellow2: [35, 204, 255]
  hsv_red1:    [0,140,100]
  hsv_red2:    [15,255,255]
  hsv_red3:    [165,140,100]
  hsv_red4:    [180,255,255]
